config.md

```diff
@@ -1343,6 +1348,7 @@ SamplingRate is the sampling rate to use when Stress Relief is activated.
 All new traces will be deterministically sampled at this rate based only on the `traceID`.
 It should be chosen to be a rate that sends fewer samples than the average sampling rate Refinery is expected to generate.
 For example, if Refinery is configured to normally sample at a rate of 1 in 10, then Stress Relief should be configured to sample at a rate of at least 1 in 30.
+If this value is set too low it will drastically increase the amount of data sent to Honeycomb, which could overwhelm the upstream queue.
```

Comment:
What does "too low" mean? 

Is "1 in 10" lower or higher than "1 in 30"? How far outside that range would be "too low"?

Reply:
I wish we had just used sampling probability or a name other than sample rate!

Maybe something more elaborate like:
> If this value is configured to send more data to honeycomb during stress relief than the normal average sampling strategy, it may overwhelm the upstream send queue and have the opposite of the desired effect.

Reply:
Used your wording @kentquirk